src/controller/users.js

```diff
@@ -1,26 +1,42 @@
 const router = require("express").Router(),
-	updateUser = require("../actions/update-user"),
-	findUser = require("../actions/find-user");
+	auth = require("../middlewares/bearer"),
+	updateUser = require("../actions/users/update-user"),
+	findUser = require("../actions/users/find-user"),
+	notLoggedIn = {
```

Comment:
Sadly, I have no prior experience with sequelize mocking, so I can't be of much help here.